### PR TITLE
Allow empty interface

### PR DESCRIPTION
**Pull request by josephdrichard**

### CODE REVIEW

**Comment by SchSeba** [comment]
Hi @josephdrichard please rebase the PR

**Comment by josephdrichard** [comment]
I still need to retest before merging this.

**Comment by josephdrichard** [comment]
testing looks good.  Ready to merge.

File: config/crd/bases/ptp.openshift.io_ptpconfigs.yaml [hunk 1]
```diff
@@ -50,7 +50,6 @@ spec:
                     ptp4lOpts:
                       type: string
                   required:
-                  - interface
```
Comment (SchSeba) [inline config/crd/bases/ptp.openshift.io_ptpconfigs.yaml:53]: you need this change also here

https://github.com/openshift/ptp-operator/tree/master/manifests/4.9

Comment (josephdrichard) [reply config/crd/bases/ptp.openshift.io_ptpconfigs.yaml:53]: done

**Pull request merged**